specification/src/chapter3.adoc

```diff
@@ -149,24 +179,35 @@ translation)
 | Requirement
 
 | SR_MMU_001
-| Either PMP/Smepmp or MTT MUST be used to protect M-mode from lower privilege
-levels.
-
-| SR_MMU_002
 | if the Sv extension is supported then 1st-stage page tables MUST used to protect
 the S-Mode Supervisor domain from accesses made by U-Mode.
 
-| SR_MMU_003
+| SR_MMU_002
 | if the H extension is supported 1st-stage and/or G-stage page tables MUST used to protect
 Supervisor domain H/S-mode from lower privilege levels.
+
+| SR_MMU_003
+| MMU MUST be used to protect resources assigned to one workload from other workloads
 |===
 
-==== MTT
+NOTE: Dependent on the use case, it may be recommended or mandatory to protect supervisor mode from lower privilege levels (with or without the H-extension). Trusted execution environments may require the use of MMU to isolate trusted applications from each other. The sPMP may be an additional or alternative mechanism.
```

Comment:
```suggestion
NOTE: Dependent on the use case, it may be recommended or mandatory to protect supervisor mode from lower privilege levels (with or without the H-extension). Trusted execution environments may require the use of MMU to isolate trusted applications from each other. The sPMP and PMP may be an additional or alternative mechanism.
```